src/plugins/world_control/WorldControl.qml

```diff
@@ -82,6 +92,24 @@ RowLayout {
    */
   property bool paused: false
 
+  /**
+   * Reset
+   */
+  RoundButton {
+    id: stopButton
+    objectName: "stopButton"
+    visible: showReset
```

Comment:
Should reset only be visible if the sim is paused?

Reply:
you might want to reset when the simulation is running